shared/libraries/native_streaming_protocol/src/native_streaming_client_handler.cpp

```diff
@@ -236,20 +236,9 @@ void NativeStreamingClientHandler::initClientSessionHandler(SessionPtr session)
     {
         LOG_W("Closing connection caused by: {}", errorMessage);
         sessionHandler.reset();
-        if (session->isOpen())
-        {
-            session->close(
-                [this](const boost::system::error_code&)
-                {
-                    connectionStatusChanged(ClientConnectionStatus::Reconnecting);
-                    tryReconnect();
-                });
-        }
-        else
-        {
-            connectionStatusChanged(ClientConnectionStatus::Reconnecting);
-            tryReconnect();
-        }
+
```

Comment:
No need to close the session?

Reply:
No need to do this explicitly, as the connection will be closed automatically when the "Session" object is destroyed.

Reply:
Ok